metrics/src/metrics.rs

```diff
@@ -12,6 +12,46 @@ use std::thread;
 use std::time::{Duration, Instant};
 use sys_info::hostname;
 
+#[macro_export]
+macro_rules! integer {
+    ($name:expr, $value:expr) => {
+        ($name, influxdb::Value::Integer($value as i64))
+    };
+}
+
+#[macro_export]
+macro_rules! string {
+    ($name:expr, $string:expr) => {
+        ($name, solana_metrics::influxdb::Value::String($string))
+    };
+}
+
+#[macro_export]
+macro_rules! field {
+    ($var:ident, $field:expr) => {
+        $var.add_field(
+                    $field.0,
+                    $field.1,
+                );
+    };
+    ($var:ident, $field:expr, $($rest:expr),+) => {
+        $var.add_field(
+                    $field.0,
+                    $field.1,
+                );
+        field!($var, $($rest),+);
+    };
+}
+
+#[macro_export]
+macro_rules! submit {
```

Comment:
rename submit! to datapoint!?  That seams to jive better with counter!

Reply:
No verb?  like `submit_datapoint', inline with `inc_counter'

Reply:
I like just plain old `datapoint!()`.  Submit feels like a given, like what else would you do with a metrics datapoint.  We don't say `print_info!()` either, it's just `info!()`.